apps/sms/views/conversation/js/conversation.js

```diff
@@ -1703,6 +1717,22 @@ var ConversationView = {
       pElement.setAttribute('data-l10n-id', 'no-attachment-text');
     }
 
+    var lateArrivalInfos = this.computeLateArrivalInfos(message, messageStatus);
+    if (lateArrivalInfos) {
+      var lateArrivalNoticeDOM = this.tmpl.lateArrivalNotice.prepare({})
```

Comment:
ah yeah, right ! how could I misread this...

OK, because I want this to land better sooner than later, let's keep it like this and file a separate bug to change the template lib.